js.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
```

Comment:
use 2 whitespaces instead of 4 in all document
https://github.com/mate-academy/style-guides/blob/master/javascript.md#whitespace--spaces